change/@ni-nimble-components-883f2b11-5389-4299-84e4-80c6f0b1d2c1.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
The next time Rick shows up to one of our team meetings, let's demo this feature for him. I recall there was a lot of debate about whether to also right align column headings and the decision was "lets start out not doing so and see how it looks" so now we have a chance to show him how it looks.

Reply:
Sure, I can certainly demo it. I'm not sure we're quite in a place to see the full effect of alignment & the headers though because we don't have menus in the column headers yet.